examples/worlds/environmental_sensor.sdf

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" ?>
-<!-- This example show cases how to load and unload environmental data. -->
+<!-- This example show cases how to use a custom sensor to sample data.
+You should be able to get data echoed on the topic
+"/world/environmental_sensor_test/model/model_with_sensor/link/link/sensor/custom_sensor/environmental_sensor/humidity"
```

Comment:
@arjo129 meta: that's a pretty lengthy path. Can it be shortened?

Reply:
Looks from the tutorial that it can be customized by `<topic>`, it would be worth adding that to this example file.